contracts/pools/PoolMigrator.sol

```diff
@@ -152,8 +154,8 @@ contract PoolMigrator is IPoolMigrator, Upgradeable, Utils {
             averageRate: data.averageRate,
             depositLimit: data.depositLimit,
             liquidity: PoolLiquidity({
-                bntTradingLiquidity: data.liquidity.bntTradingLiquidity,
-                baseTokenTradingLiquidity: data.liquidity.baseTokenTradingLiquidity,
+                bntTradingLiquidity: SafeCast.toUint128(data.liquidity.bntTradingLiquidity),
```

Comment:
pretty sure both values are uint128 here

Reply:
They are in `struct PoolLiquidity`.
However, here, the type of variable `data` is `struct PoolLiquidityV1`.

Reply:
fixed